hooks/check-commits.sh

```diff
@@ -17,17 +17,27 @@ fi
 HOOKS_DIR=${HOOKS_DIR:-${GIT_DIR}/hooks}
 tmp_msg_file=$(mktemp)
 tmp_diff_file=$(mktemp)
+failure=0
 
 finish() {
 	rm -f ${tmp_msg_file} ${tmp_diff_file}
 }
 trap finish EXIT
 
+echo "::add-matcher::${HOOKS_DIR}/check-diff-matcher.json"
```

Comment:
Do these screw with the actual githooks?

Reply:
I guess it will add a line to the output, but I don't know the effect of this line with the actual githooks (broken anyway see #11)

Reply:
Would it be possible to guard them with some env variable?

Reply:
Probably, using some GH action env variable. I'll see what I can do.